plugins/techdocs/src/reader/transformers/scrollIntoAnchor.test.ts

```diff
@@ -39,7 +39,7 @@ describe('scrollIntoAnchor', () => {
     window.location.hash = hash;
     transformer(dom as unknown as Element);
     jest.advanceTimersByTime(200);
-    expect(dom.querySelector).toHaveBeenCalledWith(`#${hash.slice(1)}`);
+    expect(dom.querySelector).toHaveBeenCalledWith(`[id='${hash.slice(1)}']`);
```

Comment:
The Anchor transformer was fixed here:
https://github.com/backstage/backstage/pull/10311

Reply:
Thank you @camilaibs . I just merged the master branch into mine!